python/processors/aptos_ambassador_token/processor.py

```diff
@@ -0,0 +1,85 @@
+import argparse
+import grpc
+import ast
+import json
+
+from aptos.indexer.v1 import raw_data_pb2_grpc
+from aptos.indexer.v1 import raw_data_pb2
+from aptos.transaction.testing1.v1 import transaction_pb2
+from utils.config import Config
+from utils.general_utils import standardize_address
+
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    "-c", "--config", help="Path to config file", required=True)
+args = parser.parse_args()
+config = Config.from_yaml_file(args.config)
+starting_version = 0
+if config.starting_version_default != None:
```

Comment:
@rtso Can all of this boilerplate config -> metadata/options conversion be done in a library we provide? This way people wouldn't need to write this every time they write a processor

Reply:
Yes agreed that stuff should def go into the general framework.

Reply:
Yeah I've started abstracting some of the config stuff into its own class but this was a special case (no db writes) that isn't supported yet that I plan to look into. Same thing with connecting to grpc and returning a stream of txns; I want to turn this into a framework with least boilerplate as possible.